dataset-spec/template.schema.json

```diff
@@ -103,8 +132,35 @@
                     "description": "spatial resolution in units of the spatial coordinate reference system",
                     "examples": [0.05, 10, 300]
                 },
-                "metadata": {
-                    "type": "object"
+                "spatial_size": {
+                    "type": ["array"],
+                    "prefixItems": [
+                        {
+                            "type": "integer",
+                            "exclusiveMinimum": 0
+                        },
+                        {
+                            "type": "integer",
+                            "exclusiveMinimum": 0
+                        }
```

Comment:
`exclusiveMinimum` is `1`, i.e. use `minimum: 2`. Otherwise we have no defined spatial resolution that can be computed from spatial coordinates.

Reply:
Makes sense, done.